cmd/kops/create_cluster.go

```diff
@@ -502,6 +505,32 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		return fmt.Errorf("Invalid topology %s.", c.Topology)
 	}
 
+	// NAT Gateway/ElasticIP
+	if c.NgwIds != "" {
+		ngwEipList := make([]string, 0)
```

Comment:
FYI, `ngwEIPList := []string{}` does the same thing, but 99% of the time you just want `var ngwEIPList string[]` because nil is treated as an empty slice by len, append and range